packages/atomic/src/global/global.pcss

```diff
@@ -22,4 +22,29 @@
   .link {
     @apply text-primary hover:text-primary-light hover:underline focus:text-primary-light focus:underline;
   }
+
+  .input-primary {
+    @apply border border-neutral bg-background rounded  hover:border-primary-light focus:border-primary focus:outline-none focus:ring-2 focus:ring-ring-primary;
+  }
+
+  .btn-primary {
+    @apply rounded border bg-primary hover:bg-primary-light focus:bg-primary-light focus:outline-none text-on-primary;
+  }
+
+  .btn-outline-primary {
+    @apply rounded border border-neutral bg-background hover:border-primary-light focus:border-primary focus:outline-none text-primary hover:text-primary-light;
+  }
+
+  .ripple {
+    @apply absolute pointer-events-none rounded-full;
+    transform: scale(0);
+    animation: ripple 700ms linear;
+  }
+
+  @keyframes ripple {
+    to {
+      transform: scale(4);
+      opacity: 0;
+    }
+  }
```

Comment:
Similar to my other comment, if we want to use this in result templates we should take this out of `global.pcss` and into its own file. `global.pcss` could import the new file.